modules/serverless-rn-1-33-0.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-33-0_{context}"]
+= Red Hat {ServerlessProductName} 1.33
+
+{ServerlessProductName} 1.33 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-33-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.12.
+* {ServerlessProductName} now uses Knative Eventing 1.12.
+* {ServerlessProductName} now uses Kourier 1.12.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.12.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.12.
+* The `kn func` CLI plugin now uses `func` 1.13.
+
+* OpenShift Serverless on ARM64 is now available as Technology Preview (TP).
+* The `NamespacedKafka` annotation is now deprecated. Use the standard Kafka broker without data plane isolation instead.
+
+* When enabling the automatic `EventType` auto-creation, you can now easily discover events available within the cluster. This functionality is available as a developer preview.
+
+
+* You can now use the Custom Metrics Autoscaler Operator to autoscale Knative Eventing sources for Apache Kafka sources (`KafkaSource`). This functionality is available as a technology preview (TP) feature, offering enhanced scalability and efficiency for Kafka-based event sources within Knative Eventing.
+
+[id="known-issues-1-33-0_{context}"]
+== Known issues
+
+* Due to different mount point permissions, direct upload on a cluster build does not work on IBM zSystems (s390x) and IBM Power (ppc64le).
+
+* Building and deploying a function using Podman version 4.6 fails with the `invalid pull policy "1"` error.
++
+To work around this issue, use Podman version 4.5.
```

Comment:
Maybe out of scope for the PR, but I see mostly across the RN assembly it's lowercase "podman", but a few recent entries (and this new one) are using init cap "Podman". Suggest syncing up one way or the other going forward.

Reply:
We'll address this in a future PR, this is too cumbersome to do without the original PR creator.